api/src/ginger_calls.rs

```diff
@@ -215,7 +247,7 @@ pub fn compute_msg_to_sign(
 ) -> Result<(FieldElement, FieldElement), Error> {
 
     let mr_bt = if bt_list.is_empty() {
-        PHANTOM_MERKLE_ROOT //Note: Can actually be taken from MHT_PARAMETERS::nodes[BT_MERKLE_TREE_HEIGHT]
+        MHT_PARAMETERS.nodes[BT_MERKLE_TREE_HEIGHT].clone()
```

Comment:
Ignore the TODO, let's stick with the PHANTOM_MERKLE_ROOT for the moment (if you see in Ginger it's generated starting from a magic string)

Reply:
In ginger-lib we have PHANTOM_MERKLE_ROOT defined only for MNT4, what the correct parameter for it for tweedle?

Reply:
I will recompute it in future, stick with this now